checkov/terraform/checks/resource/digitalocean/DropletSSHKeys.py

```diff
@@ -0,0 +1,24 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from typing import Any
+
+from checkov.common.models.consts import ANY_VALUE
+
+
+class DropletSSHKeys(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure the droplet specifies an SSH key"
+        id = "CKV_DIO_2"
+        supported_resources = ['digitalocean_droplet']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self):
+        self.evaluated_keys = ["ssh_keys"]
```

Comment:
you can remove this line, it will be automatically set with `"ssh_keys"` for a `BaseResourceValueCheck `